src/elements.rs

```diff
@@ -15,24 +15,40 @@ use webdriver::error::WebDriverError;
 /// The same goes for inspecting [element state](https://www.w3.org/TR/webdriver1/#element-state).
 #[derive(Clone, Debug, Serialize)]
 pub struct Element {
+    /// The high-level WebDriver client, for sending commands.
     #[serde(skip_serializing)]
     pub(crate) client: Client,
+    /// The encapsulated WebElement struct.
     #[serde(flatten)]
     pub(crate) element: webdriver::common::WebElement,
 }
 
-/// An HTML form on the current page.
-#[derive(Clone, Debug)]
-pub struct Form {
-    pub(crate) client: Client,
-    pub(crate) form: webdriver::common::WebElement,
-}
-
 impl Element {
+    /// Construct an `Element` with the specified element id.
+    /// The element id is the id given by the webdriver.
+    pub fn from_element_id(client: Client, element_id: String) -> Self {
```

Comment:
I wonder if we should provide a newtype for `ElementId` given how common it is in the API?

Reply:
Good idea!